src-plugins/itkProcessRegistrationDiffeomorphicDemons/itkProcessRegistrationDiffeomorphicDemons.cpp

```diff
@@ -205,22 +198,23 @@ template <typename PixelType>
     resampler->SetOutputDirection( proc->fixedImage()->GetDirection() );
     resampler->SetDefaultPixelValue( 0 );
 
-    try {
+    try
+    {
         resampler->Update();
     }
-    catch (itk::ExceptionObject &e) {
-        qDebug() << e.GetDescription();
-        return 1;
+    catch (itk::ExceptionObject & err)
```

Comment:
The try-catch block for the registration call above catches exceptions of type ```std::exception``` but this one catches the more specific ```itk::ExceptionObject```. It's safer to catch ```std::exception``` to avoid uncaught exceptions.

Reply:
Can wait for https://github.com/Inria-Asclepios/medInria-public/issues/301